specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/preview/2023-10-01-preview/cognitiveservices.json

```diff
@@ -5631,25 +5631,16 @@
       "description": "Azure OpenAI Content Filter.",
       "type": "object",
       "properties": {
-        "policyName": {
+        "name": {
           "type": "string",
           "description": "Name of Content Filter."
         },
-        "description": {
-          "type": "string",
-          "description": "Description of Content Filter."
+        "isMultiLevelFilter": {
+          "type": "boolean",
+          "description": "If the Content Filter has multi severity levels(Low, Medium, or High)."
         },
-        "filterType": {
-          "type": "string",
-          "description": "Content Filter type.",
-          "enum": [
-            "MultiLevel",
-            "Switch"
-          ],
-          "x-ms-enum": {
-            "name": "RaiContentFilterType",
-            "modelAsString": true
-          }
+        "source": {
```

Comment:
>source [](http://example.com/codeflow?start=9&length=6)

This property (and the filterType/isMultilevelFilter property) belongs in a top-level property bag, per RPC requirements: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#request---get-resource-collection-under-subscription](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#request---get-resource-collection-under-subscription).

Reply:
[ARMBlockingComment] This needs to be fixed with the current API version itself.

Reply:
We will take the suggests from https://github.com/Azure/azure-rest-api-specs/pull/29228#issuecomment-2183505277. To use "post" action instead of "get". We won't treat "RaiContentFilter"  as a ARM resource. In this case, shall we need to move these properties?